### PR TITLE
Feat: http server image

### PR DESCRIPTION
**Pull request by Aylr**
## What's here

* [x] http server scripts copied here
* [x] http server docker image
* [x] dockerfile builds http capable image

## Known issues

* secrets not gracefully handled yet

### CODE REVIEW

**Comment by tomdavidson** [comment]
This style of build reduced the image size from 1.2 gb to 568 mb (much of that was the "extras").  Adding http deps in the wemmick stage yields a 623 mb image but including it with the wheel in the build-wemmick stage is 583 mb. Not tons but difference but enough.

Since we are building one dual mode image, lets add the http stuff to the wemmick wheel.

File: Dockerfile [hunk 1]
```diff
@@ -53,8 +52,18 @@ LABEL org.opencontainers.image.title="Wemmick"
 LABEL org.opencontainers.image.description="BERA's Great Expectations runtime image"
 
 COPY --from=wemmick-build /dist/*.whl /tmp/dist/
-RUN pip install --no-index --find-links=/tmp/dist/ \
-      wemmick \
+RUN pip install --no-index --find-links=/tmp/dist/ wemmick \
       && rm -rf /tmp/dist
+COPY requirements-http.txt /tmp/
+RUN pip install -r /tmp/requirements-http.txt \
+    && rm -rf /tmp/requirements-http.txt
```
Comment (tomdavidson) [inline Dockerfile:59]: this file is already included in a lower level via ln57 - cant delete it.

**Comment by tomdavidson** [comment]
The poc image was over 1.2gb (much of that was the extras that we are not currently installing).  The multistage dockerfile we merged was around 620 mb without http deps. I thought the wheel was an bundled executable but since it is not, it just adds ~ 100mb to the image size.

Ditching the build stages and the wheel makes it faster ... and cuts the size of the wheel out. With http this one is about 540 mb.

Not sure about python but the server package nested in wemmick package seem a bit funny. I flattened it - want to be schooled if this is wrong.

Kept the http deps separate but included it in setup.py - doesnt really matter now the build stages are gone. If secret manger is needed for the server, is it needed for the cli?

**Comment by Aylr** [comment]
Nice changes to the Dockerfile Tom! While I like the idea of wheels, they definitely slowed down build time. I felt like half my day yesterday was waiting for those to build. I'll touch up a few small things after your changes and merge this.

**Pull request merged**